sites/public/src/forms/filters/FilterForm.tsx

```diff
@@ -98,9 +97,10 @@ const FilterForm = (props: FilterFormProps) => {
           controlClassName="control"
           options={availabilityOptions}
           defaultValue={props.filterState?.availability}
+          labelClassName="filter-header"
         />
-        <label className="field-label">{t("listingFilters.bedrooms")}</label>
-        <div className="flex flex-row bedroom-selector">
+        <label className="field-label filter-header">{t("listingFilters.bedrooms")}</label>
+        <div className="flex flex-col bedroom-selector">
```

Comment:
Could we use two columns here, like it is for the (soon to be submitted) accessibility features filters? Having them all aligned on the left feels inefficient, UI-wise.

Reply:
The thinking here is the mocks show 2 columns for the accessibility features after we hit more than 6 options. Since there are only 5 here I put them in 1 column. 

I've sent an email to Kai-Ting to ask for her input, but in the interest of getting something in before the mayor meeting can we revisit this once we get a response from her and submit this for now?

Reply:
SGTM!